File: app/src/main/java/com/neome/api/ent/entMain/msg/MsgAutomationStateInfoList.java
// neome.ai API - do not change
//

package com.neome.api.ent.entMain.msg;

import com.neome.api.ent.base.Types.EnumAutomationStateFilterKind;
import com.neome.api.nucleus.base.msg.Msg;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class MsgAutomationStateInfoList extends Msg
{
  @Nullable
  public EnumAutomationStateFilterKind[] filterAutomationStateSet;

  @Nullable
  public Date from;

  @Nullable
  public Long limit;
}

File: app/src/main/java/com/neome/api/ent/entMain/msg/MsgAutomationStateInfoList.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entMain.msg

import com.neome.api.ent.base.Types.EnumAutomationStateFilterKind
import com.neome.api.nucleus.base.msg.Msg

interface MsgAutomationStateInfoList : Msg
{
  val filterAutomationStateSet: List<EnumAutomationStateFilterKind>?
  val from: String?
  val limit: Long?
}